### ISSUE TITLE
nightly build for brew formulas using brew test-bot

### ISSUE THREAD

**Issue by osrf-migration** [ignition-tooling/release-tools#66]
**Original report ([archived issue](https://osrf-migration.github.io/ignition-tooling-gh-pages/#!/osrf/release-tools/issues/66)) by Jose Luis Rivero (Bitbucket: [Jose Luis Rivero](https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/)).**

----------------------------------------
Inspiration from [this @scpeters comment](https://osrf-migration.github.io/ignition-tooling-gh-pages/#!/osrf/release-tools/pull-requests/167/brew-bottles-creation-and-hash-updating/activity#comment-13739923).

Basically run `brew test-bot --tap=osrf/simulation --ci-testing <formulae>` nightly to test that the brew formulas are working fine. Specially useful since it can detect if we lack a bottle or there are some other problems.

**Comment by scpeters** [comment]
we have nightly jobs for many ignition bottles as of https://github.com/ignition-tooling/release-tools/pull/209
